hypothesis-python/tests/cover/test_direct_strategies.py

```diff
@@ -175,6 +176,13 @@ def fn_ktest(*fnkwargs):
     (ds.slices, {"size": -1}),
     (ds.slices, {"size": 2.3}),
     (ds.sampled_from, {"elements": ()}),
+    (ds.ip_addresses, {"v": "4"}),
+    (ds.ip_addresses, {"v": 4.0}),
+    (ds.ip_addresses, {"v": 5}),
+    (ds.ip_addresses, {"v": 4, "network": "::/64"}),
+    (ds.ip_addresses, {"v": 6, "network": "127.0.0.0/8"}),
+    (ds.ip_addresses, {"network": b"127.0.0.0/8"}),
+    (ds.ip_addresses, {"network": b"::/64"}),
```

Comment:
Is it worth adding a couple of tests for things that look like IP network strings but aren't?

e.g. `(ds.ip_addresses, {"network": "127.0.0.0/64"})`

Or is that what the bytestring test cases are?

Reply:
Yep, that's the bytestring cases... I've added a comment to explain this to future readers, including myself!